feat: improve the website with AI 🤖 (#495)
countr/website: content/docs/03-premium.md

# Premium

Countr offers Premium access to users who want to take their counting game to the next level. With Premium, your standard limits are increased and you get access to fancy new features.

## Premium-only Features {#features}

Upgrade to Premium and get access to these exclusive features:

> * Premium bot with less downtime, faster response time, and faster startup time during outages

Reviewer: ```suggestion
* Access to the Countr Premium bot with less downtime, faster response times, and faster startups after outages
```

* Early access to new beta features
* Premium support on our support server, including a fancy role for your support
* Live scoreboard that updates every five minutes
* Up to five counting channels
* Up to 100 position roles
* More flows, triggers, and actions (25 of each)

> Please note that these features are only available through the Premium bot and are not available through the normal bot. Upgrade to Premium today and take your counting game to the next level!

Reviewer: ```suggestion
These features are only available with the Countr Premium bot! Upgrade to Premium today and take your counting game to the next level!
```

### Add Premium to Your Server {#how-to-add}

In the support server, you'll have access to the `/server add` and `/server remove` commands. Using these commands, you can add the server(s) you want to be granted Premium access. You don't need to be the server owner or anything, and you don't even need to be in the server you want to give Premium to. Just know that if you add someone else's server, they will get access to Premium and use up one of your server slots.

After you've done that, you can ask one of the support members for an invite link. You can list all your servers by right-clicking your profile, going to Apps in the dropdown, and then clicking "See premium information". If you're unsure how to do this, please let us know.

## Refunds Policy {#refunds}

> Please note that there are no refunds for Premium. Premium is only $1/month, so refunding you will be a waste of time for both of us. If you're unsure about sacrificing a dollar to at least test out Premium's features, then you should probably just keep your money instead.

Reviewer: this is huge snarky for my taste but also girlbossing

Author: image.png

> If you change your mind later and want to downgrade to the free bot, you can safely do so by kicking the Premium bot and adding the free bot again. Both bots use the same database, so your data is safe and mirrored over when you add the bot back.

Reviewer: ```suggestion
If your subscription ends or you've changed your mind later and want or need to downgrade to the free bot, you can safely do so by kicking the Countr Premium bot and adding the free bot again. Both bots use the same database, so your data is safe and not lost during the transition. 
```
